src/components/Personalization/index.js

```diff
@@ -149,9 +133,18 @@ const createPersonalization = ({ config, logger, eventManager }) => {
 
     commands: {
       getDecisions(options = {}) {
-        const { scopes = [] } = options;
+        const { viewStart, scopes = [] } = options;
+        if (viewStart || scopes) {
+          if (viewStart) {
+            scopes.push(PAGE_WIDE_SCOPE);
+          }
+
+          return filterDecisions(decisionsStorage, scopes);
+        }
 
-        return filterDecisions(decisionsStorage, scopes);
+        throw new Error(
+          "Invalid getDecisions command options parameter. Please consult the documentation."
```

Comment:
If we can, let's be explicit here about what the problem is so they might not have to consult the documentation. Something along the lines of "viewStart must set to true or scopes must be defined"